.github/dependabot.yml

```diff
@@ -16,3 +16,8 @@ updates:
     directory: "/ui/"
     schedule:
       interval: "daily"
+
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
We need to add `package-ecosystem: "docker"` for each directory we want dependabot to scan.  Unfortunately we cannot pass multiple directories as a list.